castervoice/apps/browser/chrome_and_firefox.py

```diff
@@ -0,0 +1,98 @@
+#
```

Comment:
I would recommend renaming this to 'browser_standard'.

As people add other browsers this is the basic functionality we can expect them to implement. We can add in the comments and documentation which browsers this currently supports.

Reply:
how is BrowserSharedCommands?

Reply:
Personally I like standard better but I suppose it works.

Reply:
The standard is in browser.py not. This file is just commands that have the same hotkeys between browsers

Reply:
ok